programs/ziti-edge-tunnel/package/deb/postrm.in

```diff
@@ -13,7 +13,13 @@ if [ "$1" = "purge" ]; then
 fi
 # End copied section
 
-if [ -L @SYSTEMD_UNIT_DIR@/@SYSTEMD_UNIT_FILE_NAME@ ]; then
+# delete the symlink to the executable that was created by the postinst scriptlet
+if [ -L /usr/bin/@CPACK_PACKAGE_NAME@ ]; then
+    unlink /usr/bin/@CPACK_PACKAGE_NAME@
+fi
+
+# delete the regular file of the service unit that was copied by the postinst scriptlet
+if [ -e @SYSTEMD_UNIT_DIR@/@SYSTEMD_UNIT_FILE_NAME@ ]; then
     unlink @SYSTEMD_UNIT_DIR@/@SYSTEMD_UNIT_FILE_NAME@
```

Comment:
We must delete this file with the remove scriptlet because it is not specified in the package manifest but was created by the install scriptlet. This changes the condition to match a regular file, not a symlink, and to only attempt delete if the file exists.